lib/cosmos/script/telemetry.rb

```diff
@@ -164,37 +163,27 @@ def get_tlm_details(items)
 
     # Returns the buffer from the telemetry packet.
     def get_tlm_buffer(target_name, packet_name)
-      return $cmd_tlm_server.get_tlm_buffer(target_name, packet_name)
+      $cmd_tlm_server.get_tlm_buffer(target_name, packet_name)
     end
 
     # Subscribe to one or more telemetry packets. The queue ID is returned for
     # use in get_packet_data and unsubscribe_packet_data.
     # Usage:
     #   id = subscribe_packet_data([[target_name,packet_name], ...], <queue_size>)
     def subscribe_packet_data(packets, queue_size = CmdTlmServer::DEFAULT_PACKET_DATA_QUEUE_SIZE)
-      result = $cmd_tlm_server.subscribe_packet_data(packets, queue_size)
-      result
+      $cmd_tlm_server.subscribe_packet_data(packets, queue_size)
     end
 
     # Unsubscribe to telemetry packets. Pass the queue ID which was returned by
     # the subscribe_packet_data method.
     # Usage:
     #   unsubscribe_packet_data(id)
     def unsubscribe_packet_data(id)
-      result = $cmd_tlm_server.unsubscribe_packet_data(id)
-      result
+      $cmd_tlm_server.unsubscribe_packet_data(id)
     end
 
-    # DEPRECATED
     def get_packet_data(id, non_block = false)
-      results = $cmd_tlm_server.get_packet_data(id, non_block)
-      if Array === results and results[3] and results[4]
-        results[3] = Time.at(results[3], results[4]).sys
-        results.delete_at(4)
-      end
-      results
+      $cmd_tlm_server.get_packet_data(id, non_block)
```

Comment:
Doing this is a breaking change, please undo.   Need to add smarts at get_packet or remove from shared.